pkg/virt-controller/watch/application.go

```diff
@@ -367,6 +374,18 @@ func Execute() {
 		log.Log.Infof("CDI not detected, DataVolume integration disabled")
 	}
 
+	onOpenShift, err := clusterutil.IsOnOpenShift(app.clientSet)
+	if err != nil {
+		golog.Fatalf("Error determining cluster type: %v", err)
+	}
+	if onOpenShift {
+		log.Log.Info("we are on openshift")
+		app.routeCache = app.informerFactory.OperatorRoute().GetStore()
```

Comment:
Perhaps the `OperatorRoute` should be renamed if more than the virt-operator is using it?

Reply:
I was interpreting this as, routes managed by the operator (aka the export route) IIRC the store uses a label selector to only get routes managed by the operator.